lib/c/string.zig

```diff
@@ -5,7 +5,29 @@ const common = @import("common.zig");
 comptime {
     @export(&strcmp, .{ .name = "strcmp", .linkage = common.linkage, .visibility = common.visibility });
     @export(&strlen, .{ .name = "strlen", .linkage = common.linkage, .visibility = common.visibility });
+    @export(&strnlen, .{ .name = "strnlen", .linkage = common.linkage, .visibility = common.visibility });
     @export(&strncmp, .{ .name = "strncmp", .linkage = common.linkage, .visibility = common.visibility });
+    @export(&strchr, .{ .name = "strchr", .linkage = common.linkage, .visibility = common.visibility });
+    @export(&strchr, .{ .name = "index", .linkage = common.linkage, .visibility = common.visibility });
```

Comment:
`index` and `rindex` are part of `strings.h` and so should go into `strings.zig` (you'll need to rebase on upstream `master` for that).